src/docs/deployment/linux.md

```diff
@@ -262,6 +300,36 @@ parts:
     flutter-target: lib/main.dart # The main entry-point file of the application
 ```
 
+
+## Desktop file and icon
+: Desktop entry files are used to add an application 
+  to the desktop menu. These files specify the name and 
+  icon of your application, the categories it belongs to,
+  related search keywords and more. These files have the 
+  extension .desktop and follow the XDG Desktop Entry 
+  Specification version 1.1.
+  
+## Flutter super-cool-app.desktop example
+
+: Place the .desktop file in your Flutter
+  project under `<project root>/snap/gui/super-cool-app.desktop`.
+  **Notice**: name for icon and .desktop file must be the same as you name your app in yaml file
+For example:
+
+```desktop
```

Comment:
I'm not sure that desktop is a valid type for a code block. Perhaps it should be `terminal`?